apps/src/applab/designMode.js

```diff
@@ -947,16 +947,15 @@ function isMouseEventInBounds(mouseEvent) {
  * Sets app space clipping behavior. App space is clipped if clip == true.
  * @param {boolean} clip
  */
-function setAppSpaceClipping(clip) {
+designMode.setAppSpaceClipping = function (clip) {
```

Comment:
This works for now, since it follows the idiom used throughout designMode.js.  Eventually it'd be nice to update all the static methods in this file to the more concise `export function setAppSpaceClipping(clip) {`pattern - but don't bother in this PR.